Moderation-Policy.md

```diff
@@ -192,20 +192,47 @@ of the [Code of Conduct][].
 * Moderation of Posts authored by non-Collaborators may result in those
   non-Collaborators being Blocked temporarily or indefinitely from further
   participation in the Node.js GitHub organization.
-* One may immediately bypass the above procedures if it is clear that there is no intention to
-  collaborate in good faith. In such cases, it is permissible to immediately:
-  * remove posts
-  * lock issues/pull requests
-  * delete issues/pull requests
-  * block users
-  * report accounts to GitHub
+* It is always permitted and encouraged to report users interacting in bad
+  faith to GitHub.
+* If it is clear that there is no intention to collaborate in good faith,
+  it is possible to hide comments of non-Collaborators. In that case there is
+  an exception to the reporting requirement described above.
```

Comment:
I don't think an absence of good faith is a requirement for hiding. Collaborators should be free to hide good-faith-but-off-topic comments by non-collaborators, for example.

We probably also want to be careful about using "reporting" to mean different things. I can see reading this and thinking it refers to "reporting to GitHub" since that's what is discussed in the bullet point immediately before this one. Maybe we can refer to it as "documenting" rather than "reporting"?

```suggestion
* Collaborators may [hide](https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-disruptive-comments#hiding-a-comment)
comments by non-collaborators that are disruptive, off-topic, etc. When hiding a
non-collaborator comment, the requirements for documenting a moderation action
do not apply.
```

Reply:
I think if a collaborator hides a good faith comment there is a much higher chance of escalation and the moderation team missing context to be able to properly deal with it but I also acknowledge good faith and on-topic are both subjective